liquibase-core/src/test/groovy/liquibase/resource/PathHandlerFactoryTest.groovy

```diff
@@ -25,6 +26,45 @@ class PathHandlerFactoryTest extends Specification {
         input << [null, "proto:unsupported"]
     }
 
+    @Unroll
+    def "getResource: #path"() {
+        when:
+        def pathHandlerFactory = Scope.getCurrentScope().getSingleton(PathHandlerFactory)
+
+        then:
+        (pathHandlerFactory.getResource(path).exists()) == existsWithoutResourceAccessor
+        Scope.child(Scope.Attr.resourceAccessor, new JUnitResourceAccessor(), { ->
+            assert (pathHandlerFactory.getResource(path, true).exists()) == existsWithResourceAccessor
+        })
+
+
+        where:
+        path                                                               | existsWithoutResourceAccessor | existsWithResourceAccessor
+//        "src/test/groovy/liquibase/resource/PathHandlerFactoryTest.groovy" | true                          | true
+//        "invalid/path.txt"                                                 | false                         | false
```

Comment:
Do you want these commented out?

Reply:
No, those should get put back. Had been skipping them to debug later ones that were failing. Thanks for catching that